modules/op-customizing-pipelines-as-code-configuration.adoc

```diff
@@ -0,0 +1,30 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="customizing-pipelines-as-code-configuration_{context}"]
+= Customizing {pac} configuration 
+
+[role="_abstract"]
+To customize {pac}, cluster administrators can configure the following parameters using the `pipelines-as-code` config map in the `pipelines-as-code` namespace:
+
+.Customizing {pac} configuration
+[options="header"]
+|===
+
+| Parameter | Description | Default
+
+| `application-name` | The name of the application. For example, the name displayed in the GitHub Checks labels. | `"Pipelines as Code CI"` 
```

Comment:
Do you mean for `"Pipelines as Code CI"` to be quoted this way?

Reply:
Yes, because the default value must be written as a string, including quotes.